src/Codehard.Common/Codehard.Common.AspNetCore/Extensions/ReadFileExtension.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.AspNetCore.Http;
+
+namespace Codehard.Common.AspNetCore.Extensions;
+
+/// <summary>
+/// Read file extension.
+/// </summary>
+public static class ReadFileExtension
+{
+    /// <summary>
+    /// Read file as byte array.
+    /// </summary>
+    /// <param name="formFile"></param>
+    /// <param name="cancellationToken"></param>
+    /// <returns></returns>
+    public static async Task<byte[]> ReadFileAsync(this IFormFile formFile, CancellationToken cancellationToken)
```

Comment:
ถ้า form file มีขนาดใหญ่มาก (hundreds of MB or GB) performance และ memory consumption ของ extension ตัวนี้เป็นยังไงครับ

Reply:
ขอเปิด issues ไว้ก่อน นะครับ